target/linux/generic/hack-5.15/766-net-phy-realtek-add-led-link-select-for-RTL8221.patch

```diff
@@ -0,0 +1,69 @@
+From 4c6b89509ddaccac270a4fac9277b67eedd09b83 Mon Sep 17 00:00:00 2001
+From: Yangyu Chen <cyy@cyyself.name>
+Date: Sat, 22 Apr 2023 19:01:18 +0800
+Subject: [PATCH] net: phy: realtek: add led-link-select for RTL8221
```

Comment:
Are you planning to send this to upstream Linux?

Reply:
Yes. It should better append to [https://lore.kernel.org/lkml/30e4fb909f5fac1734562c9822e5d72b5c0497cf.1682163424.git.daniel@makrotopia.org/](https://lore.kernel.org/lkml/30e4fb909f5fac1734562c9822e5d72b5c0497cf.1682163424.git.daniel@makrotopia.org/) or send this patch after the daniel's patch merged.